test.py

```diff
@@ -9,18 +9,26 @@
 import tempfile
 import time
 
-
```

Comment:
Pep8 is a fairly well established standard throughout the company.  The DC/OS CLI as well as all the DC/OS tooling itself also adheres to pep8.  I'm fine with deviations when they make sense, but let's not add regressions.